src/NodaTime/Calendars/Wondrous/WondrousCalendarHelper.cs

```diff
@@ -0,0 +1,2204 @@
+using JetBrains.Annotations;
```

Comment:
I thought we were removing this class, as it would basically be moved into the second package?
I'm happy to refactor to move the parts that are needed for the calculator into that class - and to sort out the data generation. Is there any other need for this class?